js/apps/system/_admin/aardvark/APP/react/src/views/shards/distribution/RebalanceShards.tsx

```diff
@@ -0,0 +1,139 @@
+import { Box, Button, FormLabel } from '@chakra-ui/react';
+import { Global } from '@emotion/react';
+import { Form, Formik } from 'formik';
+import React from 'react';
+import { SwitchControl } from '../../../components/form/SwitchControl';
+import { InfoTooltip } from '../../../components/tooltip/InfoTooltip';
+import { getAdminRouteForCurrentDB } from '../../../utils/arangoClient';
+
+async function rebalanceShards(opts: {
+  maximumNumberOfMoves?: number;
+  leaderChanges?: boolean;
+  moveLeaders?: boolean;
+  moveFollowers?: boolean;
+  excludeSystemCollections?: boolean;
+  piFactor?: number;
+  databasesExcluded?: string[]
+}) {
+  const route = getAdminRouteForCurrentDB();
+  try {
+    const res = await route.put("cluster/rebalance", {
+      version: 1,
+      ...opts
+    });
+    const result = res.body.result;
+    if (result.moves.length === 0) {
+      window.arangoHelper.arangoNotification('No move shard operations were performed.');
+    } else {
+      window.arangoHelper.arangoNotification('Rebalanced shards with ' + result.moves.length + ' move operation(s).');
+    }
+    return result;
+  } catch (e: any) {
+    window.arangoHelper.arangoError(
+      "Failure",
+      `Got unexpected server response: ${e.message}`
+    );
+  }
+}
+
+export const RebalanceShards = ({refetchStats}: {refetchStats: () => void}) => {
+  return (
+    <Formik
+      onSubmit={async ({includeSystemCollections, ...opts}) => {
+        const result = await rebalanceShards({
+          excludeSystemCollections: !includeSystemCollections,
+          ...opts
+        });
+        if (result && result.moves.length > 0) refetchStats();
+      }}
+      initialValues={{
+        moveLeaders: false,
+        moveFollowers: false,
+        includeSystemCollections: false,
+      }}
+      isInitialValid={true}
+    >{({isValid, isSubmitting}) => (
+      <Form>
+      <Global
+        styles={{
+          "label": {
+            marginBottom: "unset"
+          }
+        }}
+      />
+        <Box width="100%" padding="5" background="white">
+          <Box fontSize={"lg"}>
```

Comment:
Can use Text from Chakra
```suggestion
          <Text fontSize={"lg"}>
```